src/vanilla/utils/constants.ts

```diff
@@ -1 +1 @@
-export const RESET = Symbol()
+export const RESET = import.meta.env?.MODE !== 'production' ? Symbol('RESET') : Symbol()
```

Comment:
maybe we need to annotate the type somehow?

Reply:
I'm not sure what you mean by annotate. 
For symbols there is only `unique symbol` but that is implicit so I wouldn't add it.

Reply:
How would you solve the TS error?

Reply:
this works for me:
```ts
export const RESET = Symbol(
  import.meta.env?.MODE !== 'production' ? 'RESET' : undefined,
)
```
or `""` instead of the `undefined`

Reply:
<img width="1042" alt="image" src="https://github.com/pmndrs/jotai/assets/490574/58d29e42-eed9-4238-91c0-7e9f323d5d07">
It's a bit unfortunate, but two bytes should be fine.

Reply:
```suggestion
export const RESET: unique symbol = import.meta.env?.MODE !== 'production' ? Symbol('RESET') : Symbol()
```
This is what I meant by type annotation. Doesn't it work?

Reply:
No, 

```
Type 'symbol' is not assignable to type 'unique symbol'.ts(2322)
```

also casting with `as` won't accept symbol:
```
'unique symbol' types are not allowed here
```

Reply:
Okay, let's do this then as you suggested.
```suggestion
export const RESET = Symbol(import.meta.env?.MODE !== 'production' ? 'RESET' : '')
```

Reply:
It's there, the checks passed.